pkg/maps/policymap/policymap_privileged_test.go

```diff
@@ -19,22 +19,19 @@ package policymap
 import (
 	"errors"
 	"fmt"
+	"log"
 	"os"
 	"testing"
 
 	"github.com/cilium/cilium/pkg/bpf"
 	"github.com/cilium/cilium/pkg/checker"
-	"github.com/cilium/cilium/pkg/logging"
-	"github.com/cilium/cilium/pkg/logging/logfields"
 	"github.com/cilium/cilium/pkg/policy/trafficdirection"
 	"github.com/cilium/cilium/pkg/u8proto"
 
 	"golang.org/x/sys/unix"
 	. "gopkg.in/check.v1"
 )
 
-var log = logging.DefaultLogger.WithField(logfields.LogSubsys, "map-policy")
```

Comment:
Why was this line removed? Merge gone wrong maybe?

Reply:
Variable `log` is maybe shadowing the newly introduced system package "log". It seems that elsewhere we import `"log"` as `stdlog "log"` to avoid this (e.g., [pkg/policy/rule_test.go#L21](https://github.com/cilium/cilium/blob/master/pkg/policy/rule_test.go#L21)).

Changes in this file seem unrelated to this PR, however.

Reply:
I'm not certain why it was modified, only thing I can think is that VSCode might have done it in a `goimports` or `govet`. I've removed the change so there's no difference from master.